v2/protocol/http/protocol.go

```diff
@@ -223,7 +223,6 @@ type msgErr struct {
 func (p *Protocol) ServeHTTP(rw http.ResponseWriter, req *http.Request) {
 	m := NewMessageFromHttpRequest(req)
 	if m == nil || m.ReadEncoding() == binding.EncodingUnknown {
-		http.Error(rw, fmt.Sprintf("Cannot read CloudEvent: %s", binding.ErrUnknownEncoding), http.StatusBadRequest)
```

Comment:
if it returns anything it should be in the form of a cloudevent

Reply:
@n3wscott I'd appreciate some pointer or example to set all the attributes and format correctly. Do you do that in other places?